packages/babel-generator/src/index.js

```diff
@@ -56,11 +55,16 @@ function normalizeOptions(code, opts): Format {
     jsescOption: {
       quotes: "double",
       wrap: true,
+      minimal: true,
       ...opts.jsescOption,
     },
     recordAndTupleSyntaxType: opts.recordAndTupleSyntaxType,
   };
 
+  if (format.jsonCompatibleStrings) {
```

Comment:
```suggestion
  if (opts.jsonCompatibleStrings) {
```

Reply:
Lol. I went back and forth with just doing this inline in the options, forget to fix this.